tests/integration/source-maps-option-spec.js

```diff
@@ -46,12 +46,18 @@ test.describeIntegrationTest("Generating source maps:", function() {
                 '      this.props.name\n' +
                 '    );\n' +
                 '  } });\n' +
-                '//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi90ZXN0L2ZpbGUxLmpzeCJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiOztBQUFBLElBQUksS0FBSyxHQUFHLEtBQUssQ0FBQyxXQUFXLENBQUM7O0FBQUksUUFBTSxFQUFFLGtCQUFXO0FBQUksV0FBTzs7OztNQUFZLElBQUksQ0FBQyxLQUFLLENBQUMsSUFBSTtLQUFPLENBQUM7R0FBRyxFQUFDLENBQUMsQ0FBQyJ9'
+                '//# sourceMappingURL=test-file1.js.map'

```

Comment:
I'm not familiar with the sourcemap spec, is this supposed to just be the basename, or should it include the relative path?

Reply:
it could either be a full URL (somewhere.com/some/path/theFile.js.map) or relative to where the code is.
In this case since I am putting the .map files right next to the JS (minified/compiled) code it'll look there first.  So if the JS file is at zocdoc.com/the_compiled_js/test-file.js then this sourcemapping will tell the browser (and anyone else) to look at zocdoc.com/the_compiled_js/test-file.js.map. (In prod that .map file would not be found and chrome and everyone will ignore it. In dev, it'll find it and all work.)